common/tests/crypto/testCrypto.h

```diff
@@ -23,3 +23,13 @@ namespace crypto
     int testVerifyReport();
 }
 }
+
+#define COND2ERR(b, error_message) \
```

Comment:
Why do we need a duplicated version of that macro? Can't we just move it to a header file and share?

(also see comments on other macro whether we really need the do/while block)

Reply:
I guess it can be moved to `verify-report.h`.
Not sure whether to keep the one with error message or both.

Reply:
actually, those macros are technically different:
* checks in library don't need log
* checks in tests need log

I would rather keep these macro separate since pdo crypto does not use log or SAFE_LOG -- unless you have a different solution (which does not require importing log in crypto).